.travis.yml

```diff
@@ -1,11 +1,7 @@
 ---
 rvm:
   - 2.0
-  - 2.1
-  - 2.2
-  - 2.3
-  - 2.4
-  - 2.5
```

Comment:
the `2.0` and `2.6` is sufficient? Don't we want at least `2.3`? Just asking though.

Reply:
Foreman develop runs on 2.0 (EL7) and 2.5 (Debian Buster & Ubuntu Bionic).

Reply:
Ah, ok :+1:

Reply:
Please keep 2.5 in there, as Debian and Ubuntu use it.